xbmc/Application.cpp

```diff
@@ -2356,6 +2356,15 @@ bool CApplication::OnAction(const CAction &action)
       g_windowManager.ActivateWindow(WINDOW_MUSIC_PLAYLIST);
     return true;
   }
+  if (action.GetID() == ACTION_PLAYER_PROCESS_INFO)
+  {
+    if ((m_pPlayer->IsPausedPlayback() || m_pPlayer->IsPlaying()) &&
```

Comment:
if (m_pPlayer->GetCurrentPlayer() == "VideoPlayer")

is correct because this dialog is only valid for VideoPlayer